src/react-components/avatar-editor.js

```diff
@@ -0,0 +1,247 @@
+        .reduce((o, [k, v]) => ({ ...o, [k]: v }), {})
+    };
+
+    await this.createOrUpdateAvatar(avatar);
+
+    this.props.onAvatarChanged(avatar.avatar_id);
+
+    this.setState({ uploading: false });
+    this.props.saveStateAndFinish();
+  };
+
+  fileField = (name, label, accept, disabled = false, title) => (
+    <div className={styles.fileInputRow} key={name} title={title}>
+      <label htmlFor={`avatar-file_${name}`}>
+        <div className="img-box">{this.state.avatar.files[name] && <img src={this.state.avatar.files[name]} />}</div>
+        <span>{label}</span>
+      </label>
+      <input
+        id={`avatar-file_${name}`}
+        type="file"
+        accept={accept}
+        disabled={disabled}
+        onChange={e => {
+          this.inputFiles[name] = e.target.files[0];
+          URL.revokeObjectURL(this.state.avatar.files[name]);
+          this.setState({
+            avatar: {
+              ...this.state.avatar,
+              files: {
+                ...this.state.avatar.files,
+                [name]: URL.createObjectURL(e.target.files[0])
+              }
+            }
+          });
+        }}
+      />
+    </div>
+  );
+
+  textField = (name, label, disabled) => (
+    <div>
+      <label htmlFor={`#avatar-${name}`}>{label}</label>
+      <input
+        id={`avatar-${name}`}
+        type="text"
+        disabled={disabled}
+        value={this.state.avatar[name]}
+        onChange={e => this.setState({ avatar: { ...this.state.avatar, [name]: e.target.value } })}
+      />
+    </div>
+  );
+
+  checkbox = (name, label, disabled) => (
+    <div>
+      <label htmlFor={`#avatar-${name}`}>{label}</label>
+      <input
+        id={`avatar-${name}`}
+        type="checkbox"
+        disabled={disabled}
+        value={this.state.avatar[name]}
+        onChange={e => this.setState({ avatar: { ...this.state.avatar, [name]: e.target.value } })}
+      />
+    </div>
+  );
+
+  render() {
+    if (!this.props.signedIn) {
+      return (
+        <a onClick={this.props.onSignIn}>
+          <FormattedMessage id="sign-in.in" />
+        </a>
+      );
+    }
+
+    if (!this.state.avatar) {
+      return <div>Loading...</div>;
+    }
+
+    const { advanced } = this.props;
+
+    return (
+      <div className={classNames(styles.avatarSelectorContainer, "avatar-editor")}>
+        <div className="form-body">
+          {advanced && this.textField("avatar_id", "Avatar ID", true)}
+          {advanced && this.textField("parent_avatar_id", "Parent Avatar ID")}
+          {advanced && this.textField("name", "Name")}
+          {advanced && this.textField("description", "Description")}
+          {advanced && this.checkbox("allow_remixing", "Allow Remixing")}
+          {advanced && this.checkbox("allow_promotion", "Allow Promotion")}
+          {advanced && this.fileField("glb", "Avatar GLB", "model/gltf+binary,.glb")}
+
+          {this.fileField("base_map", "Base Map", "image/*")}
```

Comment:
i suggest breaking this up so that the latter 3 fields are hidden by default and revealed via an "Advanced" link

next to the base map, we should have a link to the raw template base image. and then if you click advanced there should be links to the photoshop and substances projects. i think we can also hide the github link unless you click advanced if we wanted to keep it super simple.

as a stretch goal you could make the base map one big, centered image as a poor man's preview of the avatar, with the additional advanced maps, if enabled, being the current size below it.